src/components/Plugin/SafeSnap/CustomBlock.vue

```diff
@@ -0,0 +1,307 @@
+<template>
+  <Block
+    title="SafeSnap"
+    :icon="!loading && moduleAddress ? 'refresh' : undefined"
+    @submit="updateDetails"
+  >
+    <div class="mb-3 text-center">
+      <Label> {{ infoLabel }} </Label>
+    </div>
+    <div
+      v-if="questionDetails?.questionId && !showQuestionInfo"
+      class="mb-3 text-white text-center"
+    >
+      <Label> {{ approvalData?.decision }} </Label>
+    </div>
+    <div
+      v-if="questionDetails?.questionId && showQuestionInfo"
+      class="mb-3 p-4 text-white text-center"
+    >
+      <div>
+        <Label> {{ approvalData?.decision }} </Label>
+      </div>
+      <div class="m-4 text-center">
+        <Label> {{ approvalData?.currentBond }} </Label>
+      </div>
+      <div>
+        <Label> {{ approvalData?.timeLeft }} </Label>
+      </div>
+    </div>
+    <UiButton
+      v-if="showActionButton"
+      @click="performAction"
+      :disabled="!actionEnabled"
+      v-text="actionLabel"
```

Comment:
I think it will not change the text if you change the language while being on the proposal page. It's better to use the translate function $t() here instead of doing it here: https://github.com/snapshot-labs/snapshot/pull/376/files?file-filters%5B%5D=.json&file-filters%5B%5D=.vue#diff-ed035d7307c0caf41cd50a6f0b79a10ff8b13dfba30763a1efd6d0792b4791e0R127
```suggestion
      v-text="$t(actionLabel)"
```